systemtest/src/main/java/io/strimzi/systemtest/templates/crd/KafkaTemplates.java

```diff
@@ -53,6 +53,21 @@ public static KafkaBuilder kafkaWithMetrics(String namespaceName, String kafkaCl
 
         return kafkaBuilder;
     }
+        
+    public static KafkaBuilder kafkaWithStrimziMetricsReporter(String namespaceName, String kafkaClusterName, int kafkaReplicas) {
+        KafkaBuilder kafkaBuilder = kafka(namespaceName, kafkaClusterName, kafkaReplicas)
+            .editSpec()
+                .editKafka()
+                    .withNewStrimziMetricsReporterConfig()
+                        .withNewValues()
+                            .withAllowList("kafka_server.*")
+                        .endValues()
+                    .endStrimziMetricsReporterConfig()
+                .endKafka()
+            .endSpec();
+
+        return kafkaBuilder;
+    }
```

Comment:
Maybe more question on @strimzi/system-test-contributors ... but why do we need this method? It is called from one place and does not add much value. It can be easily inlined and make the readability of the whole test much better because you would see right away what the actual configuration is.

Reply:
Yeah I wouldn't add any new method just for the metrics reporter. If it would be used on multiple other places then fine, but having it on one place doesn't add much value.